Italian/scappare/scopri-oggetto/esamina-stanza/esamina-fogli/premi-intaglio/premi-intaglio.md

```diff
@@ -0,0 +1 @@
+Il cassetto non si apre ma, inaspettatamente, senti uno strano scricchiolio provenire dalle scaffalature della libreria alle tue spalle.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.